modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevel.testcase

```diff
@@ -60,4 +64,87 @@ definition {
 		}
 	}
 
+	@description = "LPS-159844. When a walkthrough step is defined and a step is below the fold, user can see popover is."
+	@priority = "3"
+	test CanRenderWhenStepIsBelowFold {
+		property portal.acceptance = "true";
+
+		task ("Given walkthrough enabled site-level") {
+			Walkthrough.addWalkthrough(fileName = "walkthrough_configuration_multiple_page.json");
+		}
+
+		task ("Given a page width larger than the preview window.") {
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1",
+				widgetName = "JS Walkthrough Sample");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 2");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 2",
+				widgetName = "Documents and Media");
```

Comment:
I'd try to reduce dependencies from other modules here... :thinking: 

Is there some more generic and decoupled portlet not tied to a product team?

Reply:
Yep, I can try to add a image, seems good?

Reply:
sounds good!